### PR TITLE
Completed javascript-forms exercise.

### PR DESCRIPTION
**Pull request by jerome-liwanag**
javascript-forms-submission

### CODE REVIEW

File: javascript-forms/form-events.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const $contactForm = document.getElementById('contact-form');
+$contactForm.addEventListener('submit', function(event) {
+  event.preventDefault();
+  const formData = {
+    'name': $contactForm.elements.name.value, 'email': $contactForm.elements.email.value, 'message': $contactForm.elements.message.value
+  };
+  console.log('formData:', formData);
+  $contactForm.reset();
+});
```
Comment (kurtyazdizadeh) [inline javascript-forms/form-events.js]: This section actually belongs in `form-values.js`, but the code looks good! 👍

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Just need to move code from one script file to another, see comments

File: javascript-forms/form-events.js (hunk 1 above)
Comment (jerome-liwanag) [reply javascript-forms/form-events.js]: Fixed!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved! 👍

**Pull request merged**